ilagan_andreijireh/script.js

```diff
@@ -91,4 +91,37 @@ function loadComments() {
     comments = JSON.parse(storedComments);
     displayComments();
   }
-}
\ No newline at end of file
+}
+
+document.getElementById('search-button').addEventListener('click', function() {
+  var country = document.getElementById('country-input').value;
+  fetch('https://restcountries.com/v3.1/name/' + country)
```

Comment:
### Use guard clause here when possible to catch early errors

Reference: https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g25d183194fb_0_0

You can just do this instead:
.then((response) => {
      if (!response.ok) {
        throw new Error('Country not found');
      }
      return response.json();
    })